From 2c074ab07e8572abad4e2fab205fd1656f41edcc Mon Sep 17 00:00:00 2001
From: Ranlvor <ranlvor@starletp9.de>
Date: Wed, 7 Sep 2016 22:59:01 +0200
Subject: [PATCH 2/2] statuspage: blue is the new pink

change the pink in the statuspage to blue

Cause our commuity traditionally does not like the pink color
---
 package/gluon-status-page/src/css/main.css | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/gluon-status-page/src/css/main.css b/package/gluon-status-page/src/css/main.css
index 0604802..079e567 100644
--- a/package/gluon-status-page/src/css/main.css
+++ b/package/gluon-status-page/src/css/main.css
@@ -23,7 +23,7 @@ a:hover {
 header {
   display: flex;
   padding: 0 14px;
-  background: #dc0067;
+  background: #0000ff;
   color: rgba(255, 255, 255, 0.98);
   position: absolute;
   top: 0;
-- 
1.9.1

